src/utils.cpp

```diff
@@ -1210,4 +1210,18 @@ int get_kernel_ptr_width()
   return 64;
 }
 
+std::pair<std::string, std::string> split_symbol_module(
+    const std::string &symbol)
+{
+  if (!symbol_has_module(symbol))
+    return { symbol, "" };
+
+  size_t idx = symbol.rfind(" [");
+  if (idx == std::string::npos)
+    return { symbol, "" };
+
+  return { symbol.substr(0, idx),
+           symbol.substr(idx + 2, symbol.length() - idx - 3) };
```

Comment:
What are the three symbols at the end? "] \0"? 
Consider doing something like
```suggestion
symbol.substr(idx + strlen(" ["), symbol.length() - idx - strlen("] ") - 1)
```
Compilers can inline this these days.

Reply:
In fact, it's `" []"` (to get the length of the module name). But your suggestion is still very valid, I used it. Thanks!